plugins/modules/cloud/univention/udm_dns_record.py

```diff
@@ -34,11 +34,13 @@
         description:
```

Comment:
I cannot add a comment for that line, but in `requirements:` you should add something like `ipaddress (for I(type=ptr_record))`